app/assets/stylesheets/_bootstrap_overrides.scss

```diff
@@ -36,3 +44,12 @@ label.required:after {
     @extend .badge-success;
   }
 }
+
+.container {
+  padding: 0;
```

Comment:
why?

Reply:
this is how you're ending up with this:

<img width="885" alt="Screen Shot 2019-09-11 at 12 58 27 PM" src="https://user-images.githubusercontent.com/1615322/64660776-0d031700-d496-11e9-9ef3-597d50cb233e.png">

I'm not sure what you're trying to achieve by removing the padding from the `container`